scripts/upgrade

```diff
@@ -71,19 +71,19 @@ fi
 #=================================================
 
 [[ $YNH_APP_ID == "garradin" ]] \
-    && [[ "$(cat "/opt/$app/templates/.VERSION")" != 1.2.2 ]] \
+    && [[ "$(cat "/$final_path/VERSION")" != 1.2.2 ]] \
     && ynh_die --message "It look like that you have an old Garradin install. You need first upgrade Garradin instance (id: $garradin_migrate_id) and after migrate to Paheko."
 ynh_handle_app_migration --migration_id=garradin --migration_list=garradin_migrations
 
 if [[ $migration_process -eq 1 ]]; then
     # Reload variables
-    dbname=$app
-    db_user=$app
+    #dbname=$app
+    #db_user=$app
```

Comment:
I'd go for removing the commented lines. Git allows you to roll-back or to view code from previous commits.

For example, you can use `git log -p scripts/upgrade`

Reply:
+1 we don't need such lines.